### DISCUSSION TITLE
Is there a TypeScript/JavaScript version of this?

### DISCUSSION THREAD

**Discussion by wong2** [BerriAI/litellm#108] [Q&A]
This is really useful, hope I can use it in JS!

**Comment by ghost** [comment]
Hey @wong2 not yet. But that's a great next step. Got 2 q's:

- why are you using multiple LLM providers?
- why not just use a python backend since there's a lot more support for LLM apps in python?

**Reply by satish860** [reply]
I want the typescript SDK. To answer your question

1) I feel the ecosystem around typescript like Nextjs and others help beginners get production ready systems fast.

2) Also I have a team of 30 people who I want to make them learn python just for this.

But a great library just wants a typescript SDK. I am happy to do a first cut at it.
